lib/prompts/selectPublicAppPrompt.ts

```diff
@@ -47,18 +53,26 @@ const fetchPublicAppOptions = async (
     }
     return filteredPublicApps;
   } catch (error) {
-    logError(error, { accountId });
+    if (accountId) {
+      logError(error, { accountId });
+    } else {
+      logError(error);
+    }
```

Comment:
nit: This could be `logError(error, accountId? { accountId } : undefined)`

Reply:
Addressed in Addressed in https://github.com/HubSpot/hubspot-cli/pull/1293/commits/64ec64e3b160db98474cf96fff85513b85b11095